tests/outputs/cv/scopatz.tex

```diff
@@ -346,7 +346,7 @@
       
       
         \resitem{
-            \href{https://docs.google.com/document/d/1uMAx_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
+            \href{https://docs.google.com/document/d/1uMAx\_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
```

Comment:
@scopatz what is the god behavior here?

Reply:
You should not escape the underscore in hrefs: https://tex.stackexchange.com/questions/53399/is-it-necessary-to-escape-underscores-in-href-with-dollar-signs-or